### ISSUE TITLE
add support for redis

### ISSUE THREAD

**Issue by grinish21** [ninadingole/docker-images#7]

**Comment by ninadingole** [comment]
Yeah this is something I am also really interested hopefully you will see it soon :)

**Comment by grinish21** [comment]
> Yeah this is something I am also really interested hopefully you will see it soon :)

Thank you for such a wonderful repo and nice docker stacks.
